api/controllers/resource_controller.js

```diff
@@ -11,131 +11,214 @@
 let request = require('request');
 const Resources = require('../models/resource_model');
 
+/**
+ * Returns a promise of a request to Spark api for information about the current workers 
+ * in the spark cluster.
+ * 
+ * @returns {Promise}
+ */
 function get_spark_data() {
-    let spark_api_url = "http://pacific.cs.pdx.edu:8443/json";
+    // TODO: Define host dynamically
+    let spark_api_url = "http://atlantic.cs.pdx.edu:8443/json";
```

Comment:
perfect candidate for a kubernetes service

Reply:
also since we have a config file might as well make this a config variable

Reply:
I agree but all this code is soon to be ripped out so I'm deciding to leave it for now with the intention of coming back to it later.